docs-source/content/samples/simple/domains/soa-domain/_index.md

```diff
@@ -0,0 +1,605 @@
+---
+title: "SOA domain"
+date: 2019-04-18T07:32:31-05:00
+weight: 3
+description: "Sample for creating a SOA Suite domain home on an existing PV or
+PVC, and the domain resource YAML file for deploying the generated SOA domain."
+---
+
+
+The sample scripts demonstrate the creation of a SOA Suite domain home on an
+existing Kubernetes persistent volume (PV) and persistent volume claim (PVC). The scripts
+also generate the domain YAML file, which can then be used to start the Kubernetes
+artifacts of the corresponding domain. Optionally, the scripts start up the domain,
+and SOA server pods and services based on the domain type (for example, soa, osb, soaosb, soaess and soaessosb).
+
+#### Prerequisites
+
+Before you begin, read this document, [Domain resource]({{< relref "/userguide/managing-domains/domain-resource/_index.md" >}}).
+
+The following prerequisites must be handled prior to running the create domain script:
+
+* Make sure that Kubernetes is set up in the environment. For details, see [Set Up Kubernetes]({{< relref "/userguide/overview/k8s-setup.md" >}}).
```

Comment:
Please update "Set Up Kubernetes" to "Kubernetes setup guide"

Reply:
The name of that document is "Set up Kubernetes." However, I will change the link name to Kubernetes setup guide, as you wish.